src/transaction.ts

```diff
@@ -60,6 +61,7 @@ export interface RequestOptions {
 }
 
 export interface CommitOptions {
+  requestOptions?: IRequestOptions;
```

Comment:
The `IRequestOptions` interface will contain RPC priority options and the tagging options. However `CommitOptions` should not support tagging options as a parameter.
I am suggesting either to define a separate interface with just a `priority` option or another way would be to use some ts magic

```ts
requestOptions?: Pick<IRequestOptions, 'priority'>;
```

Reply:
I think we also need to think of a different name, since we have a same name parameter in `ExecuteSqlRequest` and `BatchUpdateOptions` interfaces, in which "requestOptions" will support `priority` and a `requestTag`.

Reply:
perhaps "commitRequestOptions" and "transactionRequetOptions", any suggestions are most welcome.